### PR TITLE
Enable testing by 'jest' on CI

### LINKED ISSUE #3: Add tests
- To ensure that bug fixes and feature additions are implemented as expected.

### PR DESCRIPTION
**Pull request by Rindrics**
close #3

- I added a small test for `src/printers/text.ts` to confirm jest actually works

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+22
```
Comment (bvlion) [inline .node-version]: nits: It seems advisable to pin the minor version as well to ensure compatibility and stability.

```suggestion
22.3.0
```

**Review by bvlion — COMMENTED** [review]
I have made one comment. Please check it.

File: .node-version (hunk 1 above)
Comment (rindrics) [reply .node-version]: thanks, fixing "minor" version makes sense.

but I'm planning to  free "patch" version to apply security patch automatically.

Comment (rindrics) [reply .node-version]: pinned at `22.3`: https://github.com/route06/aws-cost-cli/pull/12/commits/a8143948e3efb25f6455a2353b8a8e92e4eed1b9

Comment (bvlion) [reply .node-version]: Sure, that sounds like a good plan.

**Review by bvlion — APPROVED** [review]
LGTM!!

**Comment by rindrics** [comment]
thx!

**Pull request merged**